### PR TITLE
Keep default prop column in group property table, let horizontal scroll bar show.

### PR DESCRIPTION
**Pull request by angelgolfer-ms**

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[e981ad1](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/e981ad105bf089ac6f41eaa80b4c09b93f924608)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 21 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[e981ad1](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/e981ad105bf089ac6f41eaa80b4c09b93f924608)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[e981ad1](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/e981ad105bf089ac6f41eaa80b4c09b93f924608)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/group-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-get.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-list.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-post-groups.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/group.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/resources/group.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C9%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901090348086170-3744%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0GeQ3ucLuErBZ0VvS6XiDV1p79W4eMQ4fHQo818jY6U%3D&st=2019-01-09T03%3A46%3A23Z&se=2019-02-09T03%3A51%3A23Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ab90572](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/ab90572c82ffce150ec132b2a4be7fa165f3713b)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 21 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ab90572](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/ab90572c82ffce150ec132b2a4be7fa165f3713b)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ab90572](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/ab90572c82ffce150ec132b2a4be7fa165f3713b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/group-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-get.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-list.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-post-groups.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/group.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/resources/group.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C9%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901090348323489-3744%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=108lfy0Je55RJyYLQLrt3mOVVRznJ3mntIp0tkwPWVY%3D&st=2019-01-09T03%3A49%3A37Z&se=2019-02-09T03%3A54%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3a72192](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/3a72192d9b8303860b3fae74ee526f36f3ec4e59)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 21 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3a72192](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/3a72192d9b8303860b3fae74ee526f36f3ec4e59)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[3a72192](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/3a72192d9b8303860b3fae74ee526f36f3ec4e59)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/group-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-get.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-list.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-post-groups.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/group.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/resources/group.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C9%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901090410531542-3744%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kuR7vEZEO4vwDmXaQBB1Xnd%2FvYLhtwWbG7L6A3SyseI%3D&st=2019-01-09T04%3A09%3A36Z&se=2019-02-09T04%3A14%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[51569c2](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/51569c202a06bc8f90b722eaaf7b59a8ca984e47)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 21 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[51569c2](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/51569c202a06bc8f90b722eaaf7b59a8ca984e47)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[51569c2](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3744/commits/51569c202a06bc8f90b722eaaf7b59a8ca984e47)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/api/group-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-get.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-list.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/api/group-post-groups.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/resources/group.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/angelgolfer-ms-grp-defprop-scrll/api-reference/v1.0/resources/group.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C12%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901120757131159-3744%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=WdcGZ6xVrhzvJAhpq9yW68EUdIXNOTZR77szcgnvlXQ%3D&st=2019-01-12T07%3A55%3A39Z&se=2019-02-12T08%3A00%3A39Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/v1.0/api/group-get.md [hunk 1]
```diff
@@ -75,48 +43,93 @@ Do not supply a request body for this method.
 If successful, this method returns a `200 OK` response code and [group](../resources/group.md) object in the response body.
 
 ## Example
-#### Request
-The following is an example of the request.
+#### Request 1
```
Comment (Lauragra) [inline api-reference/v1.0/api/group-get.md]: @angelgolfer-ms , please update to follow current guidance for multiple examples.

**Comment by Lauragra** [comment]
@angelgolfer-ms , ping. Do we still want to merge in these changes?

**Pull request closed without merging**